src/translate.js

```diff
@@ -1,14 +1,14 @@
 import curry from 'lodash.curry';
 import { connect } from 'react-redux';
 import { getP } from './selectors';
-import { isFunction, isString } from './private/utils';
+import { isFunction, isString, isObject } from './private/utils';
 
 const getDisplayName = Component => (
     Component.displayName || Component.name || 'Component'
 );
 
 const mapPolyglotToProps = polyglotScope => state => ({
-    p: getP(state, { polyglotScope }),
+    p: getP(state, polyglotScope)
```

Comment:
We should keep using an object for getP 2nd parameter so that we don't change getP's API and also so that it keeps looking like a "prop" object : 

```javascript
const mapPolyglotToProps = options => state => ({
    p: getP(state, options),
});
```